### PR TITLE
chore: dont leak password from server

### PR DESCRIPTION
**Pull request by lwhiteley**
Remove all places where passwords are returned in user objects or logged to console

### CODE REVIEW

File: src/db/mongo/users.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ exports.findUser = async function (username) {
 exports.getUsers = async function (query) {
   console.log(`Getting users for query= ${JSON.stringify(query)}`);
   const collection = await connect(usersCollection);
-  return collection.find(query).toArray();
```
[the remark this answers is not in the document]
Comment (lwhiteley) [reply src/db/mongo/users.js:12]: well  it affects all, the question is the password used when we get a list of users?

[the remark this answers is not in the document]
Comment (lwhiteley) [reply src/db/mongo/users.js:12]: and to confirm assumptions

here are all the implementations of it.

the password is never used in these scenarios, which is good

Screenshot_20241007_203507

[the remark this answers is not in the document]
Comment (lwhiteley) [reply src/db/mongo/users.js:12]: is there anything else to do here regarding the users projection?

**Review by JamieSlome — APPROVED** [review]
LGTM! 🍰

**Pull request merged**